src/main/java/io/wisetime/connector/ServerRunner.java

```diff
@@ -125,7 +125,8 @@ public Server getServer() {
     private WiseTimeConnector wiseTimeConnector;
     private ApiClient apiClient;
     private String apiKey;
-    private TemplateFormatterConfig.Builder templateConfigBuilder = TemplateFormatterConfig.builder();
+    private TemplateFormatterConfig.Builder publicTemplateConfigBuilder = TemplateFormatterConfig.builder();
+    private TemplateFormatterConfig.Builder internalTemplateConfigBuilder = TemplateFormatterConfig.builder();
```

Comment:
Patricia uses another template to create charge record

Reply:
Patricia have different content on comment/narrative for Time Registration and Budget Line record.
Budget Line record contains additional data (e.g user experience factor)